src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -430,7 +430,7 @@ quota_t cap_to_ceiling(quota_t& value, const quota_t limit) {
 /// \pre schedule.size() == surplus.size()
 void dispense_negative_deltas(
   std::vector<quota_t>& schedule,
-  quota_t& delta,
+  quota_t delta,
```

Comment:
The only actual use of the modified `delta` at the call site was in `vlog`, so there was no actual effect of this change. That was because all what `quota_balancer_update()` does it does with negative `delta` values, never making them positive (`0` at best).